profiles/apparmor.d/opt.openqa-trigger-from-obs.script.rsync.sh

```diff
@@ -76,6 +76,7 @@
     /opt/openqa-trigger-from-obs/*:*/.run*/openqa*.log w,
     /usr/bin/openqa-cli rix,
     /usr/share/openqa/lib/** r,
+    /usr/share/openqa/public/** r,
```

Comment:
I would suggest to just allow the whole directory:
```suggestion
    /usr/share/openqa/** r,
```